raiden/utils/mediation_fees.py

```diff
@@ -16,22 +19,54 @@ def ppm_fee_per_channel(per_hop_fee: ProportionalFeeAmount) -> ProportionalFeeAm
     return ProportionalFeeAmount(round(per_hop_ratio / (per_hop_ratio + 2) * 1e6))
 
 
+def adjust_proportional_fee(given_propoprtional_fee) -> ProportionalFeeAmount:
```

Comment:
I should remove this function. It's a rebase leftover.

Reply:
Reminder?

Reply:
> Should it be removed?

Well yeah I said so in the above comment:

https://github.com/raiden-network/raiden/pull/4802/files#r326051401